src/glyph.js

```diff
@@ -105,7 +105,7 @@ export class Glyph {
   }
 
   static getOutlineBoundingBox(outline, scale, x_pos, y_pos) {
-    const bboxComp = new BoundingBoxComputation(x_pos, y_pos);
```

Comment:
removing these because the some glyphs don't have origins located on their path (ie: tenuto marks) and including the origin in these cases incorrectly expands the bounding box